src/domains/auth/services.ts

```diff
@@ -0,0 +1,25 @@
+import { JWT } from "next-auth/jwt";
+import { CreateUserData } from ".";
+import { api } from "@/services/api";
+
+export async function signUp(data: CreateUserData) {
+    return await api.post("auth/register", data);
+}
+
+export async function login(email: string, password: string) {
+    return await api.post("auth/login", {
+        email,
+        password,
+    });
+}
```

Comment:
Usually you can omit both async and await in similar functions. It'll still work the same (as `api.post` returns `Promise`)

```suggestion
export function login(email: string, password: string) {
    return api.post("auth/login", {
        email,
        password,
    });
}
```